### ISSUE TITLE
Plants vs Zombies always went black screen when going into level

### ISSUE THREAD

**Issue by heydootdoot** [DS-Homebrew/nds-bootstrap#1578]
**Your console:**
DS Lite

**Launch Method:**
- Launched with WiFi R4I-SDHC 3DS RTS flashcart (www.r4i-sdhc.com), autobooted to TWiLight Menu++
- There's no SCFG settings on DS Lite TWiLight Menu++, so I can't check

**Version used:**
I'm using TWiLight Menu v25.10.0 with nds-bootstrap/b4ds v0.72.0 (release)

**Game tested/used/applicable:**
- Plants vs. Zombies (NTR-BWBE-USA)

**Expected Behavior:**
Can still went into the game or even play normally into the level (currently at level 2-5)

**Actual Behavior:**
Many errors occured as mentioned on ```Steps to reproduce``` below

### Steps to reproduce
1. Having all nds-bootstrap settings to default
2. Launch the game
3. It'll either hang on white screen, went black screen with no music after tapping "Adventure" on graveyard, hangs after tapping "Touch to Start!" with music still playing, or red screen error pops up

### nds-bootstrap.ini
```ini
[NDS-BOOTSTRAP]
USE_ROM_REGION = 1
DEBUG = 0
LOGGING = 1
MACRO_MODE = 0
SLEEP_MODE = 1
SOUND_FREQ = 0
CONSOLE_MODEL = 0
HOTKEY = 6
NDS_PATH = fat:/DS ROMs/DS Games/Plants vs. Zombies (U)(RobotKillers).nds
SAV_PATH = fat:/DS ROMs/DS Games/saves/Plants vs. Zombies (U)(RobotKillers).sav
RAM_DRIVE_PATH = sd:/null.img
GUI_LANGUAGE = en
LANGUAGE = -1
REGION = 1
DSI_MODE = 0
EXTENDED_MEMORY = 0
DONOR_SDK_VER = 0
PATCH_MPU_REGION = 0
PATCH_MPU_SIZE = 0
FORCE_SLEEP_PATCH = 0
DONORTWL_NDS_PATH = fat:/DSiWare ROMs/Pop Island - Paperfield (USA).nds
APP_PATH = fat:/DSROMS~1/DSIWARES/SHANTA~1.NDS
PRV_PATH = fat:/DSROMS~1/DSIWARES/SAVES/SHANTA~1.SAV
BOOST_CPU = 1
BOOST_VRAM = 1
GAME_SOFT_RESET = 1
AP_FIX_PATH =
CARD_READ_DMA = 1
ASYNC_CARD_READ = 0
DONORTWL0_NDS_PATH = fat:/DS ROMs/DSiWares/Dr. Mario Express (USA) (Rev 2).nds
```

### Logs
[NDSBTSRP.zip](https://github.com/DS-Homebrew/nds-bootstrap/files/11714001/NDSBTSRP.zip)

### Other notes
This game used to be played just fine without any problem, even with cheat. I think this is before b4ds been merged with nds-bootstrap since the previous builds (so far i have only tried until mid 2022 builds) still shows some bugs like red screen, freeze, and even missing sprite on some occasion

### Will you be available for future issues and/or replies?
Yes, but I can't answer right away because of other things; will reply asap when I see a new reply

**Comment by RocketRobz** [comment]
Possibly related to #1523, as certain games are known to not work correctly on DSTT-based flashcards, as well as DSTWO.

**Comment by heydootdoot** [comment]
> Possibly related to #1523, as certain games are known to not work correctly on DSTT-based flashcards, as well as DSTWO.

ah i think that might actually be it, because i try launching Sonic Colors (USA) on my WiFi R4I-SDHC 3DS RTS flashcart and it does only show white screen after i launch it. so i guess this is really flashcart specific problem or DSTT-based clone problem

**Comment by heydootdoot** [comment]
i'm really sorry that this is long overdue, this issue regarding certain games like Plants vs. Zombies (USA) and Sonic Colors (USA) not working on DSTT-based clone flashcards has been resolved since TWiLight Menu v26.0.0/nds-bootstrap v0.73.0 (release) onward, i even tried it on the latest release TWiLight Menu v26.3.0/nds-bootstrap v1.0.3 as of now and it still works normal.
so i guess this issue can be closed. thank you very much for fixing this issue on B4DS side

**Comment by RocketRobz** [comment]
Ah, nice to know!
